### ISSUE TITLE
not working on gh-pages

### ISSUE THREAD

**Issue by thestrawh8** [jekyll/minima#167]
Hi, When I cloned and tried to run locally I was able to run the theme but after deploying into github. It is not syncing with gh-pages gems. Can't I run my blog using gh-pages. I checked by forking your repo without making any changes it was throwing an error where as I could see minimal mistakes and some other themes when forked were running on gh-pages. I am new to this jekyll please don't mind if I not completely making sense. Thanks

**Comment by ashmaroli** [comment]
Hi @thestrawh8,
You can definitely run Minima on GitHub Pages. In fact, we already have a [demo](https://jekyll.github.io/minima/) running on it, live, with the most recent changes.

Either ways, since you say, you faced issues, we'll look into it. Can you post the error messages GitHub sent you when you tried to deploy?

**Comment by thestrawh8** [comment]
Thank you for responding so early, Last night when I forked and was trying
to run through gh-pages it wasn't running. But when I see in the morning it
was running .. I got a doubt I deleted and forked again now it is showing
an error. Is something to do with github people ?
I just fork and directly go to settings section for that forked repository
in github and enable github pages and then it shows this way? Am I doing
something wrong ? But locally I was able run very nicely and also the
errors from github side are not clear. I have attached all of the
screenshots below. Thanks again.
